src/kademlia/routing_table.cpp

```diff
@@ -127,7 +127,7 @@ int routing_table::bucket_limit(int bucket) const
 {
 	if (!m_settings.extended_routing_table) return m_bucket_size;
 
-	static const aux::array<int, 4> size_exceptions({{16, 8, 4, 2}});
+	static const aux::array<int, 4> size_exceptions{{{16, 8, 4, 2}}};
```

Comment:
it's a bit unfortunate.. there's no way to make the constructor transparent? did you try ``using`` the base class' constructor?